networks/suzuka/suzuka-config/src/lib.rs

```diff
@@ -0,0 +1,36 @@
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct Config {
+    pub execution_config : movement_execution_util::config::Config,
```

Comment:
```suggestion
    pub execution_config : suzuka_execution_util::config::Config,
```

Reply:
Like in monza-config, this composes the common configuration from `protocol-units/execution/aptos/util`.